app/tests/src/common/bulkSummaryRequests.js

```diff
@@ -0,0 +1,366 @@
+    'Ae2tdPwUPEZLYjtsKf6xjC5TCmD3RKS5BXMY3bhPv9P8X7QATo3dLzVLU2m',
+    'Ae2tdPwUPEZEwDwRoBzJt3WwhpdEZXzQZTMx7xEjUGRqN2TziD7dj87bTnz',
+    'Ae2tdPwUPEZL7TdN2Ugr3isi3m5UoH6nXnMZA63VuYBBsbTiqALv3UMM2bS',
+    'Ae2tdPwUPEZ1j7ShGjWmWJNmtN9neEYgMcixuguum8cidaRnGERGh8QMDGb',
+    'Ae2tdPwUPEZHszSLNV6L8qAFE1Jh7Q8u5z3h4NU3NRReAn4e47zhDgJTnRC',
+    'Ae2tdPwUPEZN3GEFq8FoMzsSchMRwjcwzUoHsy3ARHUHQQHZxYzeDXtAkir',
+    'Ae2tdPwUPEZD9a9iktQ4YRVU8tSUE5KgUWJAWcVhNrh8G3xEXbd2ZJBH7Jx',
+  ],
+  [
+    'Ae2tdPwUPEZDLthWxHrTWKYgFeoQvc8THJko7QfFBqQRLjcQnBQoqduzq6q',
+    'Ae2tdPwUPEYyWhFrZwNih7D2w1FFXrvwcwyzaCh4XCricYYdGUigCFHS7Pe',
+    'Ae2tdPwUPEZLWfGLKuiWp5m4zLBAWy1LkDaSy9ycQajApZtsaKCkJJyamSd',
+    'Ae2tdPwUPEYzWEjFxbZUzRvLJyJfbhMz1DhupW4vjVwQjFuEpFKMjoZy6Ny',
+    'Ae2tdPwUPEZGdX1Dr1xtbLfCWsusntTPNbaUnTvnP2j7dTAhxTKYGLmM1qG',
+    'Ae2tdPwUPEZ4uD3NtLDVzmpVzzL39RqK4CFHSTGnNTN5Ma8DqiUhWfFPgVy',
+    'Ae2tdPwUPEYyHsy1ZVX9QGyGos2o4xyRE6ZSYou3fmWasS5EkcmmSK1rhkx',
+    'Ae2tdPwUPEYxW34N4VVVsBPKk9XB2CqTQhsPgghjpG3UcVcgrft8YYeUT6q',
+    'Ae2tdPwUPEZ8hXwJgxGqTQnyCHRvFVBaDyvQeVHNukEcu3JXMPi9j8dMKYe',
+    'Ae2tdPwUPEZ24T2cAj6YFKTrfpZ4aUid6uMSk7GK89Gg514r52qdyvSuLSa',
+    'Ae2tdPwUPEZDdf7CvxwE9gFmWR277Pj6FmUMFHgciGdZUe82jo2w11U5Gob',
+    'Ae2tdPwUPEZG3bQYAGEQZ1eJHVAFTqHrDxVeppyT5eVBzuDbZdDS2DTw4iK',
+    'Ae2tdPwUPEZN8peUM1ELGiRVrFt5VKSctYCNq9gEWuAbzevsVBFjeowEUof',
+    'Ae2tdPwUPEZ4Mj93jxTC494UDWq6VszrVTrv6NpyuviPF5pGQdvSM1631oP',
+    'Ae2tdPwUPEZ29vSHdocM8CBxgzogumqhkVsb8iKnsbLB2wpgV4TUWCWDBkh',
+    'Ae2tdPwUPEZ8xkAfy4P6BTupRYyPNqLkiVkCktimxSKr17852wJRAQtz16M',
+    'Ae2tdPwUPEZ92XTjnqfaN4M6bmv4hiJNooWyT64RYt81uM9ToCreVWxY2VD',
+    'Ae2tdPwUPEYzKBemPWgecqrU2xUbXAwVpBqx2GPnQ1vVhmsBgabtsmWBRQF',
+    'Ae2tdPwUPEZLE1YXsaszJ1fLc4g2XtLZHJmQgbjUWZs9qg6grzwgAaRwiy7',
+    'Ae2tdPwUPEZ3nVk9jvSGjEqUW2L214T1FZuiq6X7aMDbHQ3KqkN9cCdh6nz',
+  ],
+  [
+    'Ae2tdPwUPEZ64DHikwFaJMdQNJBce9tvbABG3X9gCeaNzfF68ckKX2n2L4c',
+    'Ae2tdPwUPEZLtvX2byRkHWM2kFHJh3HwvsPefnwhmWw9mJfHuXo7FBMuyxg',
+    'Ae2tdPwUPEYz66u2V5S35o7pDxzeYfTJ31ekfzFDGECosSg7TBUMGvs8pC3',
+    'Ae2tdPwUPEZ8uLjrq8p6aWNpQTwHrV5RDzdtmbZrW72xj3kAqpsZiuvQ1t4',
+    'Ae2tdPwUPEZ3EimSircxs5JoJ9BGbvmhtQX9MV7Lq6hQC3ZBNfShPiS2xHm',
+    'Ae2tdPwUPEZAUpyN6Vq1NRSxZEHYk2J6uw2qm9bXYWQXaiRTkqZY6yhSFoe',
+    'Ae2tdPwUPEZ9ax2V8dFamdbGCUASw4cgs1Wog7UCXKNYhiL34ifNfYAHYCV',
+    'Ae2tdPwUPEZCBqBMTJyMFwrknLY1qf4wuorQw9XtNUoPHoZAD4yrsBnuerU',
+    'Ae2tdPwUPEZLYjtsKf6xjC5TCmD3RKS5BXMY3bhPv9P8X7QATo3dLzVLU2m',
+    'Ae2tdPwUPEZEwDwRoBzJt3WwhpdEZXzQZTMx7xEjUGRqN2TziD7dj87bTnz',
+    'Ae2tdPwUPEZL7TdN2Ugr3isi3m5UoH6nXnMZA63VuYBBsbTiqALv3UMM2bS',
+    'Ae2tdPwUPEZ1j7ShGjWmWJNmtN9neEYgMcixuguum8cidaRnGERGh8QMDGb',
+    'Ae2tdPwUPEZHszSLNV6L8qAFE1Jh7Q8u5z3h4NU3NRReAn4e47zhDgJTnRC',
+    'Ae2tdPwUPEZN3GEFq8FoMzsSchMRwjcwzUoHsy3ARHUHQQHZxYzeDXtAkir',
+    'Ae2tdPwUPEZD9a9iktQ4YRVU8tSUE5KgUWJAWcVhNrh8G3xEXbd2ZJBH7Jx',
+  ],
+  [
+    'Ae2tdPwUPEZ6RUCnjGHFqi59k5WZLiv3HoCCNGCW8SYc5H9srdTzn1bec4W',
```

Comment:
this seems weird - why is there a request for so many addresses? I would expect that each of the summary requests happens in a bulk of `GAP_LIMIT` addresses, but this is far more than 20